debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/ApicurioRegistryTest.java

```diff
@@ -48,7 +49,7 @@
  */
 public class ApicurioRegistryTest {
 
-    private static final String DEBEZIUM_VERSION = "1.2.0.CR1";
+    private static final String DEBEZIUM_VERSION = "1.2";
```

Comment:
That's chicken-and-egg issue here. I built the image with my core changes locally to verify the test passes. We probably would have to disable this test until the next release, containing the fix, is out.

Reply:
@gunnarmorling I recommend you to create a special tag like `1.2-devel` and push the image to the registry so the test is passing also on CI. Then there will be a follow-up task for 1.2.0.Final/1.3 to point it elsewhere.

Reply:
Done; and logged https://issues.redhat.com/browse/DBZ-2256 for reverting it.

Reply:
Pushed an update to the image as per my latest change.